inc/experiments/namespace.php

```diff
@@ -399,7 +399,11 @@ function register_post_ab_test( string $test_id, array $options ) {
 	add_action( "altis.experiments.test.winner_found.{$test_id}", $options['winner_callback'], 10, 2 );
 
 	// Set up background task.
-	if ( ( ! defined( 'WP_INSTALLING' ) || ! WP_INSTALLING ) && ! wp_next_scheduled( 'altis_post_ab_test_cron', [ $test_id ] ) ) {
+	if (
+		( ! defined( 'WP_INSTALLING' ) || ! WP_INSTALLING ) &&
+		is_admin() &&
```

Comment:
@roborourke I think the `is_admin()` can potentially make this portion of the code never run. Do you know when and how register_post_ab_test() is invoked? Does it happen based on a user action? If it does then I think that's already low-frequency enough and doesn't need any further optimization.

Reply:
Might have jumped the gun with my approval, sorry.

Reply:
It has to be called like `register_post_type()` on the `init` hook

Reply:
Ah ok, seems like it runs both in and outside of wp-admin, so should be ok I think:

https://github.com/humanmade/aws-analytics/blob/66bafa0e93dbd015ddf383c40ac46c2e0f8b303f/inc/experiments/titles/namespace.php#L18-L30